libraries/psibase/common/include/psibase/nativeTables.hpp

```diff
@@ -78,6 +78,12 @@ namespace psibase
       static constexpr uint64_t forceReplay           = uint64_t(1) << 7;
       static constexpr uint64_t allowSocket           = uint64_t(1) << 8;
       static constexpr uint64_t allowNativeSubjective = uint64_t(1) << 9;
+      static constexpr uint64_t isReplacement         = uint64_t(1) << 10;
```

Comment:
Update needed to `doc/src/specifications/data-formats/package.md`

We really need a doc somewhere that explains what each of these flags are for.